src/libtomahawk/resolvers/ScriptJob.h

```diff
@@ -64,6 +66,7 @@ public slots:
     QString m_id;
     scriptobject_ptr m_scriptObject;
     QVariantMap m_data;
+    QVariant m_data_primitive;
```

Comment:
a) poor naming, m_dataPrimitive if at all
b) you don't need it, you can make m_data a qvariant and still store the map inside. then have `m_data.toMap()` in the accessor. that said you probably want a different accessor data() and dataMap() or something for the new non-map return type